### PR TITLE
feat: modal component added

### PR DESCRIPTION
**Pull request by saifudinbadani**
Modal component added:

- please review on html and css structure.

Screen Shot 2022-03-24 at 10 46 17 am

### CODE REVIEW

**Review by lomesshh — APPROVED** [review]
clean code
nice file structure 
looks good to merge

File: components/modal/modal.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        </section>
+
+        <pre>
+            <code class="language-markup">
+                <div class="modal border-1px-solid pos-rltv z-index-three">
+                    <div class="modal-text heading-3"> This is a modal</div>
+                    <button class="modal-btn pos-absolute  m-r-1">
+                        <i class="modal-close far fa-times-circle"></i>
+                    </button>
+                </div>
```
Comment (junaidshaikh-js) [inline components/modal/modal.html:72]: Why did you add this part in the `code` element?

**Review by junaidshaikh-js — APPROVED** [review]
Good Work! Check out the comment. 

If that's is for some reason that I might be missing you can merge the PR.

File: components/modal/modal.html (hunk 1 above)
Comment (saifudinbadani) [reply components/modal/modal.html:72]: It's for rendering the code snippet on page.
